src/main/resources/i18n/messages_cs.properties

```diff
@@ -21,6 +21,8 @@ label.client.cypherdetail.solution=Řešení
 label.client.cypherdetail.verify=Ověřit
 label.client.cypherdetail.finalplacedetail=Detail vyhlášení výsledků
 label.client.cypherdetail.skipped=Tuto šifru jste přeskočili
+label.client.cypherdetail.nofinalplace=Místo vyhlášení neexistuje
```

Comment:
Nevím, jestli tohle je pro klienta smysluplná hláška.

Reply:
je vůbec nějaká smysluplná hláška pro klienta? 
"Hra ještě není připravena" například?

Reply:
Dal bych něco jako: Hru ještě nění možné zahájit. Počkejte na pokyny pořadatele.
Nebo nějakou takovou srajdu.

Reply:
je to tam :)